starter_kits/TypeScript/README.md

```diff
@@ -0,0 +1,6 @@
+# Setup
+You should have typescript installed before developing. You can do this globally with `npm install -g typescript`
+Run `npm install` to get the typescript definitions for node.
+
+# Development
+Don't forget to run `tsc` to compile the typescript sources before using the bot.
```

Comment:
Is it possible to add this step to `run_game.sh`/`run_game.bat`?

Reply:
Yes. I have updated my fork with the changes. What't the next step?